doozer/doozerlib/cli/scan_osh.py

```diff
@@ -68,16 +328,16 @@ def get_tagged_all(self, tag):
         else:
             return []
 
-    def trigger_scans(self, nvrs: list):
+    def generate_commands(self, nvrs) -> (list, None):
```

Comment:
```suggestion
    def generate_commands(self, nvrs) -> list:
```

Reply:
Would `Optional[list]` work?

Reply:
added in https://github.com/openshift-eng/art-tools/pull/141/commits/e4031289cbb0f229b3524b08cc04909ef204bc17